2021-06-18 04:03:07,745 INFO    MainThread:4037270 [internal.py:wandb_internal():91] W&B internal server running at pid: 4037270, started at: 2021-06-18 04:03:07.744599
2021-06-18 04:03:07,757 INFO    WriterThread:4037270 [datastore.py:open_for_write():80] open: /home/chengzilong/DDQN/wandb/offline-run-20210618_040306-317aai4v/run-317aai4v.wandb
2021-06-18 04:03:07,761 DEBUG   HandlerThread:4037270 [handler.py:handle_request():124] handle_request: run_start
2021-06-18 04:03:10,056 DEBUG   HandlerThread:4037270 [meta.py:__init__():39] meta init
2021-06-18 04:03:10,056 DEBUG   HandlerThread:4037270 [meta.py:__init__():53] meta init done
2021-06-18 04:03:10,057 DEBUG   HandlerThread:4037270 [meta.py:probe():210] probe
2021-06-18 04:03:10,063 DEBUG   HandlerThread:4037270 [git.py:repo():33] git repository is invalid
2021-06-18 04:03:10,063 DEBUG   HandlerThread:4037270 [meta.py:_save_pip():57] save pip
2021-06-18 04:03:10,066 DEBUG   HandlerThread:4037270 [meta.py:_save_pip():71] save pip done
2021-06-18 04:03:10,066 DEBUG   HandlerThread:4037270 [meta.py:_save_conda():78] save conda
2021-06-18 04:03:14,187 DEBUG   HandlerThread:4037270 [meta.py:_save_conda():86] save conda done
2021-06-18 04:03:14,187 DEBUG   HandlerThread:4037270 [meta.py:probe():252] probe done
